carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/common/CommonUtils.java

```diff
@@ -26,20 +26,20 @@
 
     /**
      * pause
-     * 
+     *
      * @param timeout Number
      */
     public static void pause(Number timeout) {
     	Timer.start(ACTION_NAME.PAUSE);
-        LOGGER.info(String.format("Will wait for %s seconds", timeout));
+        LOGGER.debug(String.format("Will wait for %s seconds", timeout));
         try {
             Float timeoutFloat = timeout.floatValue() * 1000;
             long timeoutLong = timeoutFloat.longValue();
             Thread.sleep(timeoutLong);
         } catch (InterruptedException e) {
             e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS1148)